themes/default/content/docs/intro/console/webhooks.md

```diff
@@ -164,9 +162,11 @@ Most payloads contain `user` and `organization` fields. `user` contains the
 identity of the user who triggered the webhook. For example, the person who initiated
 the stack update or performed the action.
 
-> The Pulumi Webhook payloads are under development, and may be changed from
-> time to time. See the [Pulumi Community Slack](https://slack.pulumi.com/) for
-> any announcements or changes.
+{{% notes "warning" %}}
+The Pulumi Webhook payloads are under development, and may be changed from
+time to time. See the [Pulumi Community Slack](https://slack.pulumi.com/) for
+any announcements or changes.
+{{% /notes %}}
```

Comment:
is this warning still accurate?

Reply:
I haven't seen any discussion about changing, but I can see why we might not want to commit to GA stability.